src/app/api/create-meme/route.ts

```diff
@@ -105,7 +116,7 @@ export async function GET(request: Request) {
         console.error('Error generating meme creation transaction payload:', error);
         const errorMessage = error instanceof Error ? error.message : 'Unknown error occurred';
         const errorStack = error instanceof Error ? error.stack : undefined;
-        
```

Comment:
Everything but this looks great!😊

Reply:
I dont get why? line 108 is now line 119 due to changes. but code remains almost same, and the space still.